src/main/java/com/mraof/minestuck/client/gui/playerStats/PlayerStatsScreen.java

```diff
@@ -93,7 +93,8 @@ public boolean reqMedium()
 	public enum EditmodeGuiType
 	{
 		DEPLOY_LIST(InventoryEditmodeScreen::new, InventoryEditmodeScreen.TITLE),
-		GRIST_CACHE(GristCacheScreen::new, GristCacheScreen.TITLE);
+		GRIST_CACHE(GristCacheScreen::new, GristCacheScreen.TITLE),
+		ATHENEUM(AtheneumScreen::new, AtheneumScreen.TITLE);
```

Comment:
I put them like that since that is how the tabs are ordered in the webcomic's sburb server gui. (Disregarding the stuff we haven't implemented yet, from left to right is the Phernalia Registry, then Grist Cache, then Atheneum.)